frontend/src/common/Header/Login.jsx

```diff
@@ -38,6 +40,9 @@ const Login = ({ handleLogin, displayProfile }) => {
 
   const handleButtonShow = () => {
     if (document.cookie) {
+      const storedPostTypes = JSON.parse(localStorage.getItem('firebaseui::rememberedAccounts'));
+      console.log(storedPostTypes);
```

Comment:
// eslint-disable-next-line no-console
if you have to have the console logs, other wise just get rid of this?